inc/network/includes-network.php

```diff
@@ -526,6 +526,12 @@ function network_step2( $errors = false ) {
 define( 'PATH_CURRENT_SITE', '<?php echo $base; ?>' );
 define( 'SITE_ID_CURRENT_SITE', 1 );
 define( 'BLOG_ID_CURRENT_SITE', 1 );
+<?php
+  $config_file_contents = ob_get_contents();
+  ob_end_clean();
+  $config_file_contents = apply_filters( 'pantheon_wp_multisite_config_contents', $config_file_contents );
```

Comment:
🔢 Same as above re: filter name.

```suggestion
  $config_file_contents = apply_filters( 'pantheon.multisite.config_contents', $config_file_contents );
```